packages/itwinui-css/src/header/header.scss

```diff
@@ -73,7 +72,7 @@
 }
 
 .iui-header-breadcrumbs {
-  @include breadcrumbs.iui-breadcrumbs($breadcrumbs-height: 100%);
+  --_iui-breadcrumbs-height: 100%;
```

Comment:
Since `.iui-header-breadcrumbs` and `.iui-breadcrumbs` both have the same specificity and both set `--_iui-breadcrumbs-height`, I think it's brittle.

Should we have a `:where(.iui-breadcrumbs)` only for setting the value of `--_iui-breadcrumbs-height` (where has 0 specificity)? cc: @iTwin/itwinui-1 

`breadcrumbs.scss`:
```diff
.iui-breadcrumbs {
-  --_iui-breadcrumbs-height: var(--iui-component-height);

  @include mixins.iui-reset;
  display: flex;
  align-items: center;
  block-size: var(--_iui-breadcrumbs-height);
}

+ :where(.iui-breadcrumbs) {
+   --_iui-breadcrumbs-height: var(--iui-component-height);
+ }
```

Reply:
That makes sense to me. I'll work on implementing it after I fix the merge conflict.

Reply:
I think it's fine as-is. We control the order of rules in `all.css`, so `iui-breadcrumbs` will always come first.

The proper fix for this would be to use internal cascade layers to set priority order. We can look into that after #1919